packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart

```diff
@@ -596,21 +597,37 @@ class SshnpdImpl implements Sshnpd {
                 'No handling for ephemeralPKType ${req.clientEphemeralPKType}');
         }
       }
-      // Connect to rendezvous point using background process.
-      // This program can then exit without causing an issue.
-      Process rv = await Srv.exec(
-        req.rvdHost,
-        req.rvdPort,
-        localPort: req.requestedPort,
-        bindLocalPort: false,
-        localHost: req.requestedHost,
-        rvdAuthString: rvdAuthString,
-        sessionAESKeyString: sessionAESKey,
-        sessionIVString: sessionIV,
-        multi: true,
-        timeout: req.timeout,
-      ).run();
-      logger.info('Started rv - pid is ${rv.pid}');
+      if (Platform.environment['SRV_INLINE'] == 'true') {
```

Comment:
Feature very useful for debugging sshnpd where it will run the srv in the same process rather than forking it